guides/source/action_mailer_basics.md

```diff
@@ -530,6 +530,8 @@ url helper.
 <%= user_url(@user, host: 'example.com') %>
 ```
 
+NOTE: `non-GET` links don't work when added to mailer templates. They are treated as `GET` links.
```

Comment:
@senny Should we mention `jQuery UJS` instead of `JavaScript`? Rest of the sentence looks good to me.